src/PointwiseFunctions/GeneralRelativity/WeylElectric.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace gr {
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the electric part of the Weyl tensor in vacuum.
+ *
+ * \details Computes the electric part of the Weyl tensor in vacuum \f$E_{ij}\f$
+ * as: \f$ E_{ij} = R_{ij} + KK_{ij} - K^m_{i}K_{mj}\f$ where \f$\R_{ij}\f$ is
```

Comment:
Add the definition "`E_{ij} = n^a n^b C_{a i b j}`, where `n` is the unit-normal to the hypersurface and `C` is the Weyl tensor consistent with the conventions in \cite (catalog paper)."